### PR TITLE
Update README for basic and basic_tunnel p4app-exercises

### PR DESCRIPTION
**Pull request by jafingerhut**
There were changes made to IPv4 addresses assigned to hosts, and many
small changes to the instructions.

Also wrapped Python calls that compile the user's program with
try/except so that the compiler error messages are less likely
to scroll off the screen due to a Python stack trace.

### CODE REVIEW

**Comment by jafingerhut** [comment]
@theojepsen Pinging you in this comment, because I could not see a way to add you as a reviewer on Github.

**Comment by jafingerhut** [comment]
Note: The p4runtime.p4app/README.md seems to be a copy of the version of basic.p4app/README.md before my proposed edits in this PR.  I have not tried to update it.  I suspect that mycontroller.py needs to be renamed to main.py in order for `make run` to work at all?

**Comment by theojepsen** [comment]
@jafingerhut your changes look good.

Yes, it looks like `p4runtime.p4app/README.md` was copied from `basic.p4app`. I think we should replace it with `exercises/p4runtime/README.md` and make the same changes.

`mycontroller.py` is the controller to be implemented as part of the exercise. `main.py` starts this controller:
https://github.com/p4lang/tutorials/blob/p4app/p4app-exercises/p4runtime.p4app/main.py#L35
So it is correct to have both files.

File: p4app-exercises/basic_tunnel.p4app/README.md [hunk 1]
```diff
@@ -28,20 +28,21 @@ rule.
...
-**Important:** We use P4Runtime to install the control plane rules. The content
-of files `sX-runtime.json` refer to specific names of tables, keys, and
+**Important:** We use P4Runtime to install the control plane rules. The
+`insertTableEntry` method calls refer to specific names of tables, keys, and
 actions, as defined in the P4Info file produced by the compiler (look for the
-file `build/basic.p4info` after executing `make run`). Any changes in the P4
+file `/tmp/p4app-logs/basic_tunnel.p4info` after executing `make run`). Any changes in the P4
 program that add or rename tables, keys, or actions will need to be reflected
-in these `sX-runtime.json` files.
+in these `insertTableEntry` method calls in `main.py`.
```
Comment (antoninbas) [inline p4app-exercises/basic_tunnel.p4app/README.md]: This paragraph doesn't seem accurate unless I'm missing something.
main.py executes mycontroller.py (towards the end of the file) as a child process, when `make run` is used. But the `insertTableEntry` calls are in mycontroller.py and can be edited there. They are not in main.py.

Comment (antoninbas) [reply p4app-exercises/basic_tunnel.p4app/README.md]: Oops my bad I was looking at the wrong exercise...

**Comment by jafingerhut** [comment]
@theojepsen @antoninbas Is there someone else we should ask to look at this before it is merged, or do we think it is OK if one of us goes ahead and merges it?

**Comment by antoninbas** [comment]
I think this is good to merge so please go ahead. I'll do the exercises this afternoon so if I feel the need to make minor edits, I'll open a PR. I had some comments on your other PR (p4runtime exercise) though.

**Pull request merged**